packages/client/modules/meeting/components/AddNewPokerTemplate.tsx

```diff
@@ -53,16 +53,17 @@ const AddNewPokerTemplate = (props: Props) => {
           id
           user {
             id
-            featureFlags {
-              noTemplateLimit
```

Comment:
Now that we've removed the template limit, we should remove this flag altogether. I've created an issue: https://github.com/ParabolInc/parabol/issues/9523